moe.clover.mm3d.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: moe.clover.mm3d
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: mm3d
+finish-args:
+  # OpenGL access
+  - --device=dri
+  # Wayland access
+  - --socket=wayland
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # File access
+  - --filesystem=host
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: mm3d
+    config-opts:
+      # specify where GLU was just installed to
+      # NOTE: "-L/app/lib" is set in LDFLAGS by flatpak-builder but mm3d configure doesn't use it for testing GL LIBS.
+      - "--with-GL-cflags=-L/app/lib"
+      # specify where Qt is in the SDK
+      - "--with-Qt-bin-dir=/bin"
+      - "--with-Qt-include-dir=/include"
+      - "--with-Qt-lib-dir=/lib"
+      - "--with-Qt-translations-dir=/translations"
```

Comment:
These relative to `/usr` right? Also what does it need in /usr/bin for Qt?

Reply:
Ah, yeah they're in /usr. There is no `/translations` symlink so using /usr/translations/ probably fixes finding Qt translations. Thanks.

It uses Qt's build tools; uic, moc, rcc, and lrelease.